framework_tvb/tvb/core/services/operation_service.py

```diff
@@ -486,6 +488,27 @@ def __expand_arguments(arguments_list, range_values, range_title):
                 result.append((kw_new, range_new))
         return result
 
+    def import_dts(self, view_model, project_id):
```

Comment:
Can't we work with the current adapters mechanism? This method seems to break the patterns

Reply:
this questions answer is related to the answer to the above question about: 
"dts_imported = self.operation_services.import_dts(view_model, project_id)"